website/pages/docs/overview/getting-started.mdx

```diff
@@ -67,6 +67,12 @@ You can use the [online playground](https://play.panda-css.com) to get a taste o
 - Inspect what panda can extract using static analysis from your code
 - Preview the statically generated `.css` files
 
+## How does it work ?
+
+<StatelyMachinePreview embedId="d30cb4f3-2279-41a6-ad95-13e72ccfe884" machineId="56c607fb-f05d-46e3-ab7d-f59342466341" />
```

Comment:
How do we resolve this `StatelyMachinePreview` element?

Reply:
done in this PR, my bad I should have mentioned in this (current) PR that this is based of that linked (https://github.com/chakra-ui/panda/pull/1047) PR